bokeh/document.py

```diff
@@ -346,6 +347,10 @@ def to_json_string(self):
             'roots' : {
                 'root_ids' : root_ids,
                 'references' : self._references_json(root_references)
+            },
+            'version' : {
+                'full' : __version__,
+                'base' : __base_version__
```

Comment:
IIRC `__base_version__` is used to help name dev builds, it it probably not of interest here. +1 to version tuples or whatever, to make things more easily parseable.